app/controllers/application_controller.rb

```diff
@@ -47,6 +48,26 @@ def add_extra_sentry_metadata
     end
   end
 
+  def smoke_testing?
+    ENV['SMOKE_TESTING_KEY'].secure_compare(request.headers['X-Smoke-Test'])
```

Comment:
Can we remove the `X-` prefix from the header, as that's [a deprecated behaviour](http://tools.ietf.org/html/rfc6648)?